src/lib.rs

```diff
@@ -1906,7 +1936,9 @@ mod tests {
         // case found in fuzz testing
         #[rustfmt::skip]
         let sample = [(0u16, 287), (2u16, 1), (9u16, 2), (10u16, 1), (53u16, 5), (61u16, 5), (127u16, 2), (232u16, 1), (255u16, 21), (314u16, 2), (317u16, 2), (384u16, 2), (511u16, 3), (512u16, 2), (1599u16, 2), (2303u16, 5), (2559u16, 2), (2568u16, 3), (2815u16, 2), (6400u16, 2), (9211u16, 2), (9728u16, 2), (10790u16, 1), (10794u16, 94), (10797u16, 2), (10999u16, 2), (11007u16, 2), (11520u16, 1), (12800u16, 4), (12842u16, 2), (13823u16, 1), (14984u16, 2), (15617u16, 2), (15871u16, 4), (16128u16, 3), (16383u16, 2), (16394u16, 1), (18167u16, 2), (23807u16, 1), (32759u16, 2)];
-        let mut f = Filter::new(400, 0.1).unwrap();
+        let template = Filter::new(400, 0.1).unwrap();
```

Comment:
Is this because of the round -> ceil change?

Reply:
Right: bumped default fingerprint size, so this grabs pre-change size then rebuilds via with_fingerprint_size + 3 remainder bits.